react-ticket-booking-app/src/pages/concerts/concert-info.js

```diff
@@ -0,0 +1,67 @@
+import React, { useContext, useEffect, useState } from "react";
+import { ConcertService } from "../../services/concert-service";
+import { Button } from "style-components";
+import { useLocation } from "react-router-dom";
+import { CartContext } from "../../contexts/cart-context";
+import "./concert-info.css"
+import Datetime from "../../components/date-time";
+
+export default function ConcertInfo() {
+    var location = useLocation()
+    var id = location.state.concertId
+    var [concert, setConcert] = useState()
+    var concertService = new ConcertService()
+    const {addToCart} = useContext(CartContext)
+    useEffect(() => {
+        if (!id) return;
+
+        concertService.getConcert(id)
+            .then(data => setConcert(data.data))
+            .catch((error) => alert(error.response.data.error))	
+    }, [id])
+
+    const handleAddToCart = (item) => {
+        addToCart(item)
+    }
+
+    return (
+        <div className="concert-info">
+            <div className="primary-container">
+                <div className="main-info">
+                    <div style={{fontSize: '24px',}}>{concert?.concertName}</div>
```

Comment:
inline style it bad practise... move to class or you can use bootstrap or smth another lib